docs/makeup-listbox-button/index.html

```diff
@@ -324,7 +324,60 @@ <h3>Unselected, Floating label</h3>
             </div>
           </div>
         </div>
-        
+
+        <h3>Unselected - with descripton</h3>
+
+        <div class="listbox-button" data-makeup-auto-select="false">
+          <button
+            class="btn btn--form"
+            style="min-width: 175px"
+            aria-expanded="false"
+            aria-haspopup="listbox"
+            data-listbox-button-prefix="Color: "
+          >
+            <span class="btn__cell">
+              <span class="btn__text">Color: -</span>
+              <svg class="icon icon--chevron-down-12" focusable="false" height="10" width="14" aria-hidden="true">
+                <use xlink:href="../icons.svg#icon-chevron-down-12"></use>
+              </svg>
+            </span>
+          </button>
+          <div class="listbox-button__listbox" hidden>
+            <div class="listbox-button__options" role="listbox">
+              <div class="listbox-button__option" role="option">
+                <span class="listbox-button__value">Red</span>
```

Comment:
Let's also add the clipped punctuation on these examples.

Reply:
Done! Good catch